docker/candlepin-base-cs9/setup-supervisord.sh

```diff
@@ -1,77 +0,0 @@
-#! /bin/bash
-
-set -e
-
-source /root/dockerlib.sh
-
-setup_supervisor() {
-    dnf install -y supervisor
-    mkdir -p /var/log/supervisor
-    mkdir -p /etc/supervisor/conf.d
-    cat > /etc/supervisord.conf <<SUPERVISOR
-[supervisord]
-nodaemon=false
-logfile=/var/log/supervisor/supervisord.log
-pidfile=/var/run/supervisord.pid
-
-[include]
-files=/etc/supervisor/conf.d/*.conf
-
-[supervisorctl]
-serverurl=unix:///var/run/supervisor.sock
-
-[unix_http_server]
-file=/var/run/supervisor.sock
-
-[rpcinterface:supervisor]
-supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
-SUPERVISOR
-    cat > /etc/supervisor/conf.d/rsyslog.conf <<RSYSLOG_SUPERVISOR
-[program:rsyslog]
-command=/sbin/rsyslogd -n -c 5
-RSYSLOG_SUPERVISOR
-}
-
-setup_tomcat() {
-    cat > /etc/supervisor/conf.d/tomcat.conf <<TOMCAT_SUPERVISOR
-[program:tomcat]
-user=tomcat
-environment=CATALINA_PID="/tmp/tomcat.pid",JAVA_HOME="/usr/lib/jvm/jre"
-command=/usr/libexec/tomcat/server start
-stopsignal=TERM
-redirect_stderr=true
-TOMCAT_SUPERVISOR
-}
-
-setup_ssh() {
```

Comment:
does removing this means we can't ssh into the container (`docker exec -it`)? If so, that's a problem because need to be able to debug when issues come up

Reply:
you still can

Reply:
Example 
"""
podman-compose -p cs9test -f docker-compose-postgres-cs9.yml up
"""

Then you can enter it in new terminal with:

"""
podman exec -it <container_ID> /bin/bash
"""

Reply:
alright, then lets remove it